src/teuthology_api/models/presets.py

```diff
@@ -0,0 +1,11 @@
+from sqlmodel import Field, SQLModel, UniqueConstraint
+
+
+class Presets(SQLModel, table=True):
+    id: int = Field(primary_key=True)
+    username: str = Field(index=True)
+    name: str
+    suite: str
+    cmd: str
```

Comment:
I wonder if this could be a type of `SuiteArgs`? If not in db, we can at least verify `cmd` is of type `SuiteArgs` when receiving requests and then convert it to string when storing in db?

Reply:
yes i think that would be a better approach for validating the schema. should we use something like
```python
suite: Union[SuiteArgs, str]
```

Reply:
Update: we had a conversation on slack to go the idea of creating a different schema for requests where `"cmd"` is of type "SuiteArgs" and database `"cmd"` can stay `str`.